app/src/components/ReleaseModal.tsx

```diff
@@ -21,78 +20,114 @@ export default function ReleaseModal({ open = true, onClose, id, resourceType }:
     resourceType: resourceType
   });
   const ctx = trpc.useContext();
+
   const deleteMutation = trpc.draft.deleteDraft.useMutation({
-    onSuccess: () => {
-      notifications.show({
-        title: `Draft ${resource?.resourceType} released!`,
-        message: `Draft ${resource?.resourceType}/${resource?.id} successfully released to the Publishable Measure Repository!`,
-        icon: <CircleCheck />,
-        color: 'green'
-      });
-      ctx.draft.getDraftCounts.invalidate();
-      ctx.draft.getDrafts.invalidate();
+    onSuccess: data => {
+      console.log(`Successfully delete ${data.resourceType}/${data.draftId} from the database.`);
     },
     onError: e => {
       console.error(e);
+    }
+  });
+
+  const releaseChildMutation = trpc.service.releaseChild.useMutation({
+    onSuccess: (data, variables) => {
+      if (data.status !== 201) {
+        console.error(data.status);
+        console.log(data.status);
+        notifications.show({
+          title: `Release Failed!`,
+          message: `Server unable to process request`,
+          icon: <AlertCircle />,
+          color: 'red'
+        });
+      } else {
+        notifications.show({
+          title: `Draft ${variables.resourceType} released!`,
+          message: `Child draft ${variables.resourceType}/${data.id} successfully released to the Publishable Measure Repository!`,
+          icon: <CircleCheck />,
+          color: 'green'
+        });
+        ctx.draft.getDraftCounts.invalidate();
+        ctx.draft.getDrafts.invalidate();
+
+        // delete draft child artifact from the draft database now that it has been released
+        deleteMutation.mutate({
+          resourceType: variables.resourceType,
+          id: data.id
+        });
+      }
+    },
+    onError: (e, variables) => {
+      console.error(e);
       notifications.show({
         title: `Release Failed!`,
-        message: `Attempt to release ${resourceType} failed with message: ${e.message}`,
+        message: `Attempt to release ${variables.resourceType} failed with message: ${e.message}`,
         icon: <AlertCircle />,
         color: 'red'
       });
```

Comment:
As mentioned in our earlier discussion, we could add more detail to this error message in regard to which child artifact failed so that the user can easily navigate to it and know that it is a child artifact of the parent